Integrations/integration-ThreatExchange.yml

```diff
@@ -98,13 +98,13 @@ script:
         }
         var ec = {};
         ec.dbotScore = [];
-        ec['File/Malicious'] = [];
+        ec[outputPaths.file] = [];
         var md = tblToMd('ThreatExchange Hash Reputation', data, argToList(args.headers));
         for (var i=0; i<data.length; i++) {
             var dbotScore = 0;
             if (data[i].status == 'MALICIOUS') {
                 dbotScore = 3;
-                ec['File/Malicious'].push({MD5: data[i].md5, SHA1: data[i].sha1, SHA256: data[i].sha256});
+                addMalicious(ec,outputPaths.file,{MD5: data[i].md5, SHA1: data[i].sha1, SHA256: data[i].sha256, Malicious: {Vendor: 'ThreatExchange', Description: data[i].description}});
```

Comment:
This will not work because it will override in `ec` instead of adding. See how VT does it now.